### ISSUE TITLE
No Menu Appears when Portal 2 Launches

### ISSUE THREAD

**Issue by KeepJ96** [ValveSoftware/portal2#158]
When I launch the game using portal2.sh (I can't launch the game from the Steam Client), the game opens (after I had to rename the 'portal2' folder to 'hl2'), and I am presented with Portal 2's Main Menu... except, there is no menu. I see the background and the Portal logo, but there is no menu. No options, no sounds indicating the options are there when I try to navigate with my keyboard, nothing.

To note, I see this in the Terminal window:

```
 failed to dlopen /home/jacob/.local/share/Steam/SteamApps/common/Portal 2/portal2_linux/bin/game_shader_dx6.so error=/home/jacob/.local/share/Steam/SteamApps/common/Portal 2/portal2_linux/bin/game_shader_dx6.so: cannot open shared object file: Not a directory
 failed to dlopen /home/jacob/.local/share/Steam/SteamApps/common/Portal 2/portal2_linux/bin/game_shader_dx6.so error=/home/jacob/.local/share/Steam/SteamApps/common/Portal 2/portal2_linux/bin/game_shader_dx6.so: cannot open shared object file: Not a directory

 failed to dlopen /home/jacob/.local/share/Steam/SteamApps/common/Portal 2/portal2_linux/bin/game_shader_dx7.so error=/home/jacob/.local/share/Steam/SteamApps/common/Portal 2/portal2_linux/bin/game_shader_dx7.so: cannot open shared object file: Not a directory
 failed to dlopen /home/jacob/.local/share/Steam/SteamApps/common/Portal 2/portal2_linux/bin/game_shader_dx7.so error=/home/jacob/.local/share/Steam/SteamApps/common/Portal 2/portal2_linux/bin/game_shader_dx7.so: cannot open shared object file: Not a directory

 failed to dlopen /home/jacob/.local/share/Steam/SteamApps/common/Portal 2/portal2_linux/bin/game_shader_dx8.so error=/home/jacob/.local/share/Steam/SteamApps/common/Portal 2/portal2_linux/bin/game_shader_dx8.so: cannot open shared object file: Not a directory
 failed to dlopen /home/jacob/.local/share/Steam/SteamApps/common/Portal 2/portal2_linux/bin/game_shader_dx8.so error=/home/jacob/.local/share/Steam/SteamApps/common/Portal 2/portal2_linux/bin/game_shader_dx8.so: cannot open shared object file: Not a directory

 failed to dlopen /home/jacob/.local/share/Steam/SteamApps/common/Portal 2/portal2_linux/bin/game_shader_dx9.so error=/home/jacob/.local/share/Steam/SteamApps/common/Portal 2/portal2_linux/bin/game_shader_dx9.so: cannot open shared object file: Not a directory
 failed to dlopen /home/jacob/.local/share/Steam/SteamApps/common/Portal 2/portal2_linux/bin/game_shader_dx9.so error=/home/jacob/.local/share/Steam/SteamApps/common/Portal 2/portal2_linux/bin/game_shader_dx9.so: cannot open shared object file: Not a directory
```

I don't know if that specifically has anything to do with the missing menus, but I figured it might be pertinent.

Running Ubuntu 13.10 with an NVidia GeForce gt430.

**Comment by SleepyAkubi** [comment]
Why did you rename it? You should have tried "-game portal2" and made a bug
report if the same issue occurred.

**Comment by davidw-valve** [comment]
What do you mean by you "can't" launch the game from within the Steam client? Why not? What happens if you try?

**Comment by KeepJ96** [comment]
I renamed the folder because the game wouldn't launch if I didn't. Leaving the folder with the default name made the game spit out:

```
Setup file 'GameInfo.txt' doesn't exist in subdirectory 'hl2'.
Check your -game parameter or VCONFIG setting.
Setup file 'GameInfo.txt' doesn't exist in subdirectory 'hl2'.
Check your -game parameter or VCONFIG setting.
```

That's what led me to rename portal2 to hl2.

And absolutely nothing happens when I try to launch the game from Steam. The window pops up briefly saying "Preparing to Launch Portal 2 Beta [Beta]", then closes and nothing happens after that.

**Comment by davidw-valve** [comment]
Can you run Steam from the console and paste in the console output when you try to run Portal 2? 

I assume you have opted into the beta?

Have you tried deleting Portal 2 and redownloading it and running with a fresh install?

**Comment by KeepJ96** [comment]
Actually... I tried what func-boncey suggested (after undo-ing the name change to the folder); It worked. My apologies. It also fixed the issue with Steam not launching the game.

Though, in my defense, the game initially didn't launch through steam. I had to go through and set some system variable, $LD_LIBRARY I believe, and that lead me to a big fight with vbsp_linux... I don't know specifically what I did to make the game work.

Either way, sorry for wasting your time. I guess sometimes there's just no excuse for stupid.
